test/units/TEST-74-AUX-UTILS.keyutil.sh

```diff
@@ -51,7 +51,7 @@ testcase_pkcs7() {
     echo -n "test" > /tmp/payload
     openssl dgst -sha256 -sign /tmp/test.key -out /tmp/payload.sig /tmp/payload
     /usr/lib/systemd/systemd-keyutil --certificate /tmp/test.crt --output /tmp/payload.p7s --signature /tmp/payload.sig pkcs7
-    openssl smime -verify -binary -inform der -in /tmp/payload.p7s -content /tmp/payload -certfile /tmp/test.crt -nointern -noverify > /dev/null
```

Comment:
Actually I'd rather keep it for the detached case, as IIRC the kernel will use the cert in the keyring, not the embedded one - you can actually build the p7s without the certificate embedded and it still works

Reply:
ah ok, i'll update it to test both, since both ways should work (i.e. the internal certificate should be the same as the external cert)